test/Swashbuckle.AspNetCore.SwaggerGen.Test/XmlComments/XmlMethodInfoExtensionsTests.cs

```diff
@@ -9,20 +11,20 @@ public class XmlMethodInfoExtensionsTests
 {
     [Theory]
     [MemberData(nameof(DifferentMethodsSignatures))]
-    public void DifferentMethodsSignatures_ShouldBeExpected(MethodInfo methodInfo)
+    public void DifferentMethodsSignatures_ShouldBeExpected((MethodInfo MethodInfo, IEnumerable<string> ExpectedParameterNames, Type ExpectedReturnParameterType) theoryData)
```

Comment:
This doesn't need to be a tuple, `TheoryData` has variants with more type parameters, so it can be `TheoryData<MethodInfo, string[], Type>`.